Code samples page expanding on mduo13 work

jakeatdocforce opened the pull request:
Filter not functional.

mDuo13 committed:
Move code samples to language folders consistently

mDuo13 committed:
First draft code samples page

mDuo13 committed:
Code samples: parse READMEs better and use smarter default titles

mDuo13 committed:
Code samples: READMEs and smarter parsing

mDuo13 committed:
Code samples: update includes for moved samples

jakeatdocforce committed:
Testing sidebar filter

jakeatdocforce committed:
Testing moving back sidebar to left nav

jakeatdocforce committed:
updated CSS

jakeatdocforce committed:
Add filter for code lang

jakeatdocforce committed:
center contribute section

jakeatdocforce committed:
Updated bottom area with lines

jakeatdocforce commented:
@mDuo13  still work in progress but wasn't sure if the draft PR would trigger a preview link.

jakeatdocforce committed:
Update link-checker-pr.yml

mDuo13 commented:
Updated the preview at https://xrplf.github.io/xrpl-dev-portal/pr-preview/code_samples_page/code-samples.html

mDuo13 committed:
Code samples: cleanup for publishing

mDuo13 committed:
Update CSS for code samples rebased

mDuo13 commented:
- Implemented the filter using radio boxes rather than checkboxes (this was much less work)
- Fixed merge conflicts
- Rebased on latest master branch

mDuo13 committed:
CSS update

mDuo13 committed:
Revert "Update link-checker-pr.yml"

This reverts commit ff0bbfcffab74a1a8a6150d7ebd8b59ee5c5a229.

mDuo13 committed:
Code samples: copy edit descriptions

mDuo13 committed:
Code samples cleanup:

- Move Quickstart to a language folder
- Copy-edits to code sample blurbs/titles

mDuo13 committed:
Code samples: sort intro things first

mDuo13 committed:
Code Samples: change title
